src/ui/components/TestSuite/views/TestItem/TestCaseSectionRow.tsx

```diff
@@ -79,7 +79,9 @@ export function TestCaseSectionRow({
       data-status={status}
       data-type={testStep.type}
       data-test-name="TestCaseSectionRow"
-      onClick={() => startTransition(() => selectTestStep(testStep))}
+      onClick={() => {
+        selectTestStep(testStep);
+      }}
```

Comment:
> Prior to this PR, clicking on a list item often took two clicks on the same list item somehow. It still seems a bit inconsistent, but it's more reliably jumping to the right line on a single click now.

I wish we had dug into _why_ this required two clicks before. I wouldn't expect a change like this to "fix" behavior like you're describing. Sounds like it maybe just changed the timing of it but left it around still.

I think it's best to try updating values like this in transitions in case any views suspend in response to the change. (This can cause React to unmount already-mounted components so it can show their fallback states.)

I don't know if we'll see that happening here or not but it's something to look out for after this change.